x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/calculate_rule_source_from_asset.ts

```diff
@@ -42,7 +44,11 @@ export const calculateRuleSourceFromAsset = ({
     };
   }
 
-  const isCustomized = calculateIsCustomized(assetWithMatchingVersion, rule);
+  const isCustomized = calculateIsCustomized({
+    baseRule: assetWithMatchingVersion,
+    nextRule: rule,
+    isRuleCustomizationEnabled,
+  });
```

Comment:
This function also has this block:

```ts
  if (assetWithMatchingVersion == null) {
    return {
      type: 'external',
      is_customized: true,
    };
  }
```

Probably we should calculate this based on the feature flag too?

Reply:
Yes, let's add a check there as well for consistency 👍